src/test/java/uk/gov/companieshouse/efs/api/categorytemplates/service/CategoryTemplateServiceImplTest.java

```diff
@@ -128,7 +128,7 @@ void testFetchEntityAndMapToCategoryTemplateListApiObjectByCategory() {
     }
 
     @ParameterizedTest
-    @CsvSource({"RP,'','',OTHER", "CC,'','',CC", "IA1986,INS,'',INS"})
+    @CsvSource({"RP,'','',RP", "CC,'','',CC", "IA1986,INS,'',INS"})
```

Comment:
We'll need a test where the result is `OTHER`, so best to add a separate RP test results in `RP`:

```@CsvSource({"SH,'','',OTHER", "RP,'','',RP", "CC,'','',CC", "IA1986,INS,'',INS"})```

Reply:
PS. I'm guessing the SH category result is `OTHER`, please test and change to another category if necessary.

Reply:
good point, though looking again the set of enumerations now covers all 6 of our existing top level categories.  So I don't think 'OTHER' will be ever return.
(there is a `SHARE_CAPITAL("SH")` constant)
I suppose it would be appropriate to add this to the test though, to cover any new category that might be added?:
`"DUMMY,'','',OTHER",`

Reply:
Good idea 👍